buildSrc/src/main/groovy/airbyte-source-acceptance-test.gradle

```diff
@@ -19,7 +19,7 @@ class AirbyteSourceAcceptanceTestPlugin implements Plugin<Project> {
                         '-w', "$targetMountDirectory",
                         '-e', "AIRBYTE_SAT_CONNECTOR_DIR=${project.projectDir.absolutePath}",
                         'airbyte/source-acceptance-test',
-                        '-p', 'integration_tests.acceptance',
+                        '-p', 'source_acceptance_test.plugin',
```

Comment:
I don’t understand a little, a change in this line leads to the fact that we can delete the acceptance.py and integration_test.py files, but still use the SAT tests?

Reply:
yes, you don't need acceptance.py, all test cases will be picked up by plugin according to acceptance test config